appcues/src/main/java/com/appcues/ui/primitive/ExperiencePrimitive.kt

```diff
@@ -90,10 +91,11 @@ internal fun BoxScope.BackgroundImage(style: ComponentStyle) {
                 modifier = Modifier.matchParentSize(),
                 model = context.getImageRequest(imageUrl, contentMode),
                 contentDescription = null,
-                imageLoader = context.getImageLoader(),
+                imageLoader = LocalImageLoader.current ?: context.getImageLoader(),
                 placeholder = context.blurHashPlaceholder(decodedBlurHash, intrinsicSize),
                 contentScale = contentMode.toImageAsyncContentScale(),
-                alignment = getBoxAlignment(horizontalAlignment, verticalAlignment)
+                alignment = getBoxAlignment(horizontalAlignment, verticalAlignment),
+                error = context.blurHashPlaceholder(decodedBlurHash, intrinsicSize),
```

Comment:
@andretortolano oh meant to call out - this was a minor fix i noticed in the tests - when an image fails to load, we'll now just continue to show the blurHash, matching iOS. (same thing line 28 in ImagePrimitive below)

Reply:
nice catch, didnt thought of that